src/components/sbb-icon/request.ts

```diff
@@ -0,0 +1,43 @@
+import { validateContent } from './validate';
+
+export const iconNamespaces = new Map<string, string>([
+  ['sbb', 'https://lyne-icons.netlify.app/icons/'],
+]);
+export const cachedIcons = new Map<string, string>();
+export const registeredIcons = new Map<string, string>();
+const requests = new Map<string, Promise<any>>();
+
+export const getSvgContent = (url: string, sanitize: boolean): Promise<any> => {
```

Comment:
Just a thought; Should we maybe return the loaded icon content after line 24? This would allow this function to be directly used to load an icon and not have to additionally use the cachedIcons object. This would mean changing the return value to `Promise<string>`.